packages/cli/test/Prompt.test.ts

```diff
@@ -0,0 +1,49 @@
+import * as Prompt from "@effect/cli/Prompt"
+import * as MockTerminal from "@effect/cli/test/services/MockTerminal"
+import type { Terminal } from "@effect/platform"
```

Comment:
```suggestion
import type { Terminal } from "@effect/platform/Terminal"
```

To keep imports consistent.